core/primitives/src/bandwidth_scheduler.rs

```diff
@@ -313,8 +315,9 @@ impl BlockBandwidthRequests {
     ProtocolSchema,
 )]
 pub struct BandwidthSchedulerState {
-    /// Random data for now
-    pub mock_data: [u8; 32],
+    pub link_allowances: Vec<((ShardId, ShardId), u64)>,
```

Comment:
nit: Make the inner type into a struct please `struct LinkAllowance` 

q: Why vector?